core/services/ocr2/plugins/median/services.go

```diff
@@ -167,7 +167,7 @@ func NewMedianServices(ctx context.Context,
 			abort()
 			return
 		}
-		median := loop.NewMedianService(lggr, telem, cmdFn, medianProvider, dataSource, juelsPerFeeCoinSource, gasPriceSubunitsDataSource, errorLog)
+		median := loop.NewMedianService(lggr, telem, cmdFn, medianProvider, dataSource, juelsPerFeeCoinSource, errorLog)
```

Comment:
This change is irrelevant to the `ChainWriter` work, however updating the `chainlink-common` dependency caught this error.

Reply:
Are you sure you aren't downgrading chainlink-common? IIRC This is a brand new argument

Reply:
I did accidentally, yes. It's been added back.